### DISCUSSION TITLE
tool/onedrive/webdav

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#279] [Announcements]
# tool/onedrive/webdav

import WebDav from 

https://alist.nn.ci/tool/onedrive/webdav.html

**Comment by Chellane** [comment]
OneDrive: https://xx-my.sharepoint.xx/personal/xx_xx_xx/_layouts/15/onedrive.aspx

**Reply by Chellane** [reply]
这链接是从哪里可以获取到？
我直接把网页OneDrive的链接放进去，生成的webdav仍然无法使用

**Reply by xuanxuan1231** [reply]
OneDrive需要是企业版的（比如E5），个人版的好像不行

**Reply by kis766** [reply]
@chellane 你的onedrive 成功了吗？我的跟你一样的问题

**Comment by istek** [comment]
onedrive 个人版也是可以使用webdav的，你们可以看看这个网址；https://winscp.net/eng/docs/guide_microsoft_onedrive

**Reply by kis766** [reply]
你的这个是要另下一个软件才能挂载onedrive 吧 ，用alist的话不行啊
